fakestorage/object.go

```diff
@@ -113,14 +116,17 @@ func toBackendObjects(objects []Object) []backend.Object {
 func fromBackendObjects(objects []backend.Object) []Object {
 	backendObjects := []Object{}
 	for _, o := range objects {
+		var acl []storage.ACLRule
+		_ = json.Unmarshal(o.ACL, &acl)
```

Comment:
🤔 not sure if ignoring this error is a very good idea. If o.ACL is not empty, it should be a valid acl.

Alternatively, we can let the backend handle this and make sure that backend.Object also has ``[]storage.ACLRule``. Thoughts?

Reply:
we could return an error here but it would break the pattern. Can you show me what you mean by letting the backend handle it?

Reply:
I thought about making it ``[]storage.ACLRule`` in backend.Object, then implementing {Marshal,Unmarshal}JSON to properly convert it to a byte slice before marshaling. Not sure if that would work though.

If you prefer, I can merge this as is and give it a shot. Let me know what you think.